clang/lib/CodeGen/CMakeLists.txt

```diff
@@ -99,4 +99,5 @@ add_clang_library(clangCodeGen
   clangFrontend
   clangLex
   clangSerialization
+  clangSema
```

Comment:
Can you add these in alphabetical order here and in other files? Looks like that is how it is maintained.

Reply:
Thanks. Done.